src/components/Header/Header.js

```diff
@@ -20,7 +37,9 @@ export const Header = () => {
               </a></li>
 
               <li><a href="#">
-                <img src="../../icons/crop-header-icons-off-white/crop-hazelnut2x.png" alt="Crop Icon" width="35" height="35"/>
+                <img onClick={cycleThrough.bind(this, cropImageName)}

```

Comment:
Doesn't need to be changed here since the impact is minimal, but for your reference its actually more efficient to have a class function that calls this.props.cycleThrough and have that class function bound to this in the constructor of this class.